core/src/main/java/bisq/core/app/BisqSetup.java

```diff
@@ -547,6 +551,18 @@ private void checkForInvalidMakerFeeTxs() {
         });
     }
 
+    private void checkFreeDiskSpace() {
+        long REQUIRED_FREE_DISK_SPACE = 2147483648L;
```

Comment:
Renaming `REQUIRED_FREE_DISK_SPACE` to something like `2GbInBytes` would make the code easier to read.